### PR TITLE
rest login

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: app/src/main/java/fr/sharpbunny/emergencytag/HttpHandler.java [hunk 1]
```diff
@@ -17,13 +17,12 @@
...
 import java.net.URLEncoder;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 import javax.net.ssl.HttpsURLConnection;
 
 public class HttpHandler {
-
+    // TAG is used in log to identify origin of log
```
Comment (viasisco) [inline app/src/main/java/fr/sharpbunny/emergencytag/HttpHandler.java]: Use a true javadoc syntax

File: app/src/main/java/fr/sharpbunny/emergencytag/HttpHandler.java [hunk 2]
```diff
@@ -17,13 +17,12 @@
...
     private static final String TAG = HttpHandler.class.getSimpleName();
 
     public HttpHandler() {
```
Comment (viasisco) [inline app/src/main/java/fr/sharpbunny/emergencytag/HttpHandler.java]: javadoc?

File: app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java [hunk 3]
```diff
@@ -63,84 +53,80 @@ public void Login(View v) {
 
         login = ChampLogin.getText().toString();
         password = ChampPassword.getText().toString();
-
-        new GetLogin().execute();
+        
+        // hack to login with rest without typing anything
```
Comment (viasisco) [inline app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java]: no comment...

File: app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java [hunk 4]
```diff
@@ -63,84 +53,80 @@ public void Login(View v) {
...
-
-                    // Getting JSON Array node
-                    JSONArray response = jsonObj.getJSONArray("contacts");
-
-                    // make use of response here
-
-
-                    //
-
+                    // Getting JSON message
```
Comment (viasisco) [inline app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java:126]: you can refer to key status. 0 if ok and 1 or more if not.

**Review by viasisco — CHANGES_REQUESTED** [review]

File: app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java (hunk 4 above)
Comment (frederic34) [reply app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java:126]: done

File: app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java (hunk 3 above)
Comment (frederic34) [reply app/src/main/java/fr/sharpbunny/emergencytag/LoginActivity.java]: no response....

**Review by Fuzhyon — APPROVED** [review]

**Pull request merged**